### DISCUSSION TITLE
Carattere del registro elettronico

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#714] [Idee e proposte]
Gentile @trinko,
potrei avere delle indicazioni per comprendere come modificare il carattere della sezione registro, in particolare Argomenti/Attività? 
Purtroppo il carattere è veramente piccolo.
Attendo sua risposta.

**Comment by trinko** [comment]
Si può fare in diversi modi.
Un modo è modificare i fogli degli stili CSS, presenti in **public/css**.

Un altro modo, più semplice, è modificare i template. Ad esempio, per la modifica su Argomenti/attività:
- copia il file **templates/lezioni/registro_firme_G.html.twig** nella cartella **PERSONAL/templates/lezioni/registro_firme_G.html.twig**
- modifica quindi il file appena copiato; 
In questo modo non sarà sovrascritto quando fai un aggiornamento.

Per la modifica in questione, nel file indicato si può aggiungere la proprietà _style_ con il comando _font-size_.
Oppure, dovrebbe essere sufficiente rimuovere il tag _SMALL_ nella riga 202 nel modo seguente:
```
 {{ (v.argomenti|first)|raw }}
```

**Comment by luigiadoc** [comment]
Se vado in PERSONAL/templates/lezioni/registro_firme_G.html.twig non mi risulta nulla

**Reply by trinko** [reply]
> Se vado in PERSONAL/templates/lezioni/registro_firme_G.html.twig non mi risulta nulla

Come dicevo, devi copiare il file **templates/lezioni/registro_firme_G.html.twig** nel nuovo percorso  **PERSONAL/templates/lezioni/registro_firme_G.html.twig**

**Comment by luigiadoc** [comment]
perfetto, sono riuscita. Grazie
